docs/specs/RIPR-SPEC-0159-bounded-helper-transfer.md

```
@@ -0,0 +1,128 @@
+# RIPR-SPEC-0159: Bounded helper-call transfer
+
+Status: proposed
+
+Issue: #3296 (parent #3215; builds on #3294 / RIPR-SPEC-0157 and
+#3295 / RIPR-SPEC-0158)
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)